pipeline/liminal.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+name: GettingStartedPipeline
+volumes:
+  - volume: gettingstartedvol
+    claim_name: gettingstartedvol-pvc
+    local:
+      path: .
+images:
+  - image: mmrs-python
+    type: python
+    source: .
+pipelines:
+  - pipeline: getting_started_pipeline
+    owner: Bosco Albert Baracus
+    start_date: 1970-01-01
+    timeout_minutes: 10
+    schedule: 0 * 1 * *
+    default_arg_loaded: check
+    default_array_loaded: [2, 3, 4]
+    default_object_loaded:
+      key1: val1
+      key2: val2
+    metrics:
+      namespace: TestNamespace
+      backends: [ ]
+    tasks:
+      - task: mmrs_download
+        type: python
+        description: download mmrs data
+        image: mmrs-python
```

Comment:
How was this image built?

Reply:
``` yaml
 images
  - image: mmrs-python
    type: python
    source: 
```

The build of images is triggered by liminal build. It used the directive images in liminal.yml like seen above. It seems that as I specified python type, liminal automatically install libs from requirements.txt and as source is the current directory ,it is added inside the docker image. However i don't deep dive because it is my first time to use the tool and choose it because it's easy to setup/run for rapid test.

Reply:
From their doc, wher's the build process: https://github.com/apache/incubator-liminal/tree/master/liminal/build/image/python